distributed/deploy/tests/test_ssh.py

```diff
@@ -32,7 +32,12 @@
         ["127.0.0.1"] * 3,
         connect_options=dict(known_hosts=None),
         asynchronous=True,
-        worker_options={"nthreads": 2, "memory_limit": "2 GiB", "death_timeout": "5s"},
+        worker_options={
+            "nprocs": 2,
```

Comment:
Perhaps an inline comment here to explain that `nprocs` is an option from `cli/dask_worker.py` and why that is significant would be helpful. It is a rather subtle change. Perhaps an assertion below that uses it would also help.